cloud

```diff
@@ -1725,6 +1742,7 @@ function execute_command() {
     eval "declare -a argv=($argv)"
 
     declare -r load_queue_trigger_topic="${prefix}load_queue_trigger"
+    declare -r storage_bucket="${project}${prefix}cache"
```

Comment:
Please add `_` between `${project}` and `${prefix}`. This will make the bucket name easier to read.